public/app/plugins/datasource/elasticsearch/components/ElasticsearchQueryField.tsx

```diff
@@ -26,7 +26,7 @@ class ElasticsearchQueryField extends React.PureComponent<Props, State> {
     ];
 
     this.state = {
-      syntaxLoaded: false,
+      syntaxLoaded: true,
```

Comment:
Seems like change not related to the PR.

Reply:
Oh good catch, leftover from when I was helping someone debug an issue